tips-and-tricks/rolling_averages_with_dask.ipynb

```diff
@@ -0,0 +1,2047 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set Up A Cluster\n",
+    "\n",
+    "This is going to employ a three worker CPU machine cluster, so we can handle some large data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[2020-11-19 19:08:52] INFO - dask-saturn | Cluster is ready\n"
+     ]
+    }
+   ],
+   "source": [
+    "from dask_saturn import SaturnCluster\n",
+    "from dask.distributed import Client\n",
+    "\n",
+    "cluster = SaturnCluster(\n",
+    "    scheduler_size='medium',\n",
+    "    worker_size='xlarge',\n",
+    "    n_workers=3,\n",
+    "    nthreads=4,\n",
+    ")\n",
+    "client = Client(cluster)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table style=\"border: 2px solid white;\">\n",
+       "<tr>\n",
+       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
+       "<h3 style=\"text-align: left;\">Client</h3>\n",
+       "<ul style=\"text-align: left; list-style: none; margin: 0; padding: 0;\">\n",
+       "  <li><b>Scheduler: </b>tcp://d-steph-ml-workshop-443217fa1282437da4d84713212eacde.main-namespace:8786</li>\n",
+       "  <li><b>Dashboard: </b><a href='https://d-steph-ml-workshop-443217fa1282437da4d84713212eacde.internal.saturnenterprise.io' target='_blank'>https://d-steph-ml-workshop-443217fa1282437da4d84713212eacde.internal.saturnenterprise.io</a></li>\n",
+       "</ul>\n",
+       "</td>\n",
+       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
+       "<h3 style=\"text-align: left;\">Cluster</h3>\n",
+       "<ul style=\"text-align: left; list-style:none; margin: 0; padding: 0;\">\n",
+       "  <li><b>Workers: </b>3</li>\n",
+       "  <li><b>Cores: </b>12</li>\n",
+       "  <li><b>Memory: </b>94.50 GB</li>\n",
+       "</ul>\n",
+       "</td>\n",
+       "</tr>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<Client: 'tcp://10.0.11.197:8786' processes=3 threads=12, memory=94.50 GB>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client.restart()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import s3fs\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import dask\n",
+    "import dask.dataframe as dd\n",
+    "from dask.distributed import wait\n",
+    "import datetime\n",
+    "import warnings\n",
+    "warnings.simplefilter(\"ignore\")"
```

Comment:
I think this warnings-filtering should be dropped. See this explanation from @jsignell : https://github.com/saturncloud/examples/pull/41#pullrequestreview-531827222

Reply:
Oh ok- yeah it's just a thing that got copied in from another notebook, I can cut it.